build.gradle

```diff
@@ -355,14 +356,23 @@ allprojects {
             }
           }
         }
+
+        maven {
+          name = "GitHubPackages"
+          url = "https://maven.pkg.github.com/consensys/tuweni"
+          credentials {
+            username = System.getenv('GITHUB_ACTOR')
+            password = System.getenv('GITHUB_TOKEN')
+          }
+        }
       }
       publications {
         MavenDeployment(MavenPublication) { publication ->
           if (project != rootProject) {
             from components.java
             artifact sourcesJar { classifier 'sources' }
           }
-          groupId 'io.tmio'
+          groupId 'io.consensys'
```

Comment:
do we want this to be io.consensys.protocols ?

Reply:
yes, we should use new group id.